_playwright-tests/UI/PopularRepo.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { test, expect } from '@playwright/test';
+import { navigateToRepositories } from './helpers/navHelpers';
+import { closePopupsIfExist } from './helpers/helpers';
+import { deleteAllPopularRepos } from './helpers/deletePopularRepositories';
+
+test.describe('Popular Repositories', () => {
+  test('Add popular repos', async ({ page }) => {
+    // Ensure no popular repos are selected.
+    await deleteAllPopularRepos(page);
+
+    await navigateToRepositories(page);
+    await closePopupsIfExist(page);
+    await expect(page).toHaveTitle('Repositories - Content | RHEL');
+   
+    await test.step('Select the Popular repos tab', async () => {
+      await page.getByRole('link', { name: 'Popular repositories' }).click();
+      await expect(page.getByTestId('popular_repos_table')).toBeVisible();
+    });
+
+    await test.step('Select and add EPEL 9', async () => {
+      await page
+        .getByRole('row', { name: 'EPEL 9 Everything x86_64' })
+        .getByLabel('Select row 0', { exact: true })
+        .click();
+      await page.getByTestId('add-selected-dropdown-toggle-no-snap').click();
+      await page.getByTestId('add-selected-repos-no-snap').click();
```

Comment:
If this would be `await page.getByRole("button", {name: "Add 1 repository without snapshotting" }).click()` it would both read better but also would test that we show the text correctly (e.g. it shows `1` and then we could have another test that would test it shows `2`).

Also the test could first verify that the button is disabled, then it is enabled with one and shows `1` and with two shows `2`...

But that is good as a followup :)

Reply:
I have updated that section to click the menu button based on its name and role.